site/dat/wiki/ParameterizedController.wiki

```diff
@@ -1,4 +1,4 @@
-= Parameterized Controller =
+= Parameterized Controller & Sampler =
```

Comment:
That's not enough for documenting it. I'd accept having both in same wiki page, but we need some explanation what each does.